### ISSUE TITLE
Use the native Windows notifications

### ISSUE THREAD

**Issue by Pawloland** [BiGilSoft/WindowTop#125]
Please add an option to use the native Windows notifications.

**Comment by gileli121** [comment]
@Pawloland 
The whole point of this request is to see notification history here:
![image](https://user-images.githubusercontent.com/17680514/128607827-e6760f7f-7182-4379-aad0-e5da0724f4f3.png)
Right?

What kind of notifications do you want to add to add?
I assume that you don't want to add notifications like this 
![image](https://user-images.githubusercontent.com/17680514/128607837-d854a125-fc6c-4feb-a75c-3a215cf7d034.png)

That you see when you enable TOP mode via a hotkey.
You want it for things like update notifications right?

**Comment by Pawloland** [comment]
You are right. I would like to be able to see all the notifications of the windowtop in the Windows built-in notification area shown in your SS.

I would be pleased even if every notification of windowtop (that is now showed using it's custom notifications), was just in this case redirected to the Windows notifications.

Now that I think about it, you're right. It makes more sense to show just some of them.
The perfect solution in my opinion would be a switch to enable/disable Windows notifications, because not everyone likes them and a list of the types of notifications in windowtop settings. 
You could enable/disable some of the notifications that windowtop sends, like for example enable crash notifications, disable notifications sent after doing specific keycombo, or disable notifications about keycombo that can't be registered, because another program is already using it. Any possible notification would have an option to be disabled.

 I know that something like it is natively supported in Android. Idk if Windows has something like this built into the OS, so it may be needed to handle it inside an windowtop settings.

Given how you implemented the current windowtop's notification system it may or may not be hard to add those 2 new functionalities, so I don't push for the second one of those.

For now having an option to just see all of the notifications in the Windows' tray would be a sufficient solution.

**Comment by gileli121** [comment]
@Pawloland 

OK, it is a good idea when the notification is relevant also for the future.
Things like
* "A new version is available"
* "WindowTop crashed, click here to report dump/logs/issue"
* "Failed to register hot-key XX because it used by another program"

Actually, it is only these 3 options ^
I don't see more good examples of such notifications.
Do you have more examples?

Anyway, it is planed for the next major release (v5.6.x)

**Comment by Pawloland** [comment]
I would add to that one which says that it can't check whether there are any updates available (it triggers most often when I'm offline). Nothing else comes to to my mind. I would especially like these new notifications to be possible to be enabled / disabled completely - one by one, so that when a user doesn't like some specific notifications but likes the others, they could be turned off, so that they won't be send, bu the rest of them will still be delivered to a user.

Thanks.

**Comment by gileli121** [comment]
@Pawloland 
Thanks for the suggestion. 
I will implement it in the feature.

Another thing, I just finished implementing a cool feature in the Shrink option when you enabling mouse interact mode and live preview. I released a build that you can use and test. Maybe you will find it useful.
You can find more info and the build here:
https://github.com/BiGilSoft/WindowTop/issues/104 

I will very much appreciate it if you can test it and give feedback on #104 
If you have feedback, please reply in #104 and not here (here it is off-topic)

**Comment by gileli121** [comment]
This feature request will be integrated with #151.
Fatal crash errors will be also windows notifications.
When you click on the notification, it will suggest you report the error.

**Comment by gileli121** [comment]
@Pawloland 
I made a mistake. it was too long ago. I forgot my argument here:

> @Pawloland
> 
> OK, it is a good idea when the notification is relevant also for the future. Things like
> 
> * "A new version is available"
> * "WindowTop crashed, click here to report dump/logs/issue"
> * "Failed to register hot-key XX because it used by another program"
> 
> Actually, it is only these 3 options ^ I don't see more good examples of such notifications. Do you have more examples?
> 
> Anyway, it is planed for the next major release (v5.6.x)

So I should implement it one day. 
Reopened.
